mmc-google-analytics/src/test/unit/java/info/novatec/metricscollector/google/MetricsTest.java

```diff
@@ -0,0 +1,45 @@
+package info.novatec.metricscollector.google;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+
+/**
+ * Created by DBE on 25.08.2017.
+ */
+public class MetricsTest {
+
+    Metrics metrics;
+
+    @Before
+    public void init(){
+        metrics = new Metrics();
+    }
+
+    @Test
+    public void addDimensionsTest(){
+        Map<String, String> expectedDimensions = new HashMap<>();
+        expectedDimensions.put("dimensionName1", "dimension1");
+        expectedDimensions.put("dimensionName2", "dimension2");
+        expectedDimensions.forEach( (String name, String value) ->
+            metrics.addDimension(name, value)
+        );
+        assertThat(metrics.getDimensions()).isEqualTo(expectedDimensions);
+    }
+
+    @Test
+    public void addMetricsTest(){
+        Map<String, Double> expectedMetrics = new HashMap<>();
```

Comment:
expectedMetrics can be moved up in class scope (same as above comment).

Reply:
same as expectedDimensions